rimage/image_processing.go

```diff
@@ -508,3 +508,51 @@ func EdgeHysteresisFiltering(mag *mat.Dense, low, high float64) (*image.Gray, er
 	}
 	return edges, nil
 }
+
+// ImageToYCbCrForTesting converts an image to YCbCr. It is only to be used for testing.
+func ImageToYCbCrForTesting(dst *image.YCbCr, src image.Image) {
```

Comment:
I couldn't find another was to convert to YCbCr, so I exported this function. Let me know if you have another idea!

Reply:
you can probably use the draw.Draw function to do this
```
func MakeYCbCr(pic image.Image) *image.YCbCr {
     result := image.NewYCbCr(pic.Bounds())
    draw.Draw(result, result.Bounds(), pic, pic.Bounds().Min, draw.Src)
    return result
```

Reply:
It doesn't look like I can do this, since image.YCbCr doesn't implement draw.Image (missing method Set).

Reply:
oh wow, yea it looks like YCbCr isn't pixel addressable! So, yea, this function seems like the best bet for now

Reply:
Just found an alternative - but if you would like to stick to using one set of images (the png sources) then what you have now seems like the way to go